qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -113,6 +128,7 @@ jobs:
   - do:
     - aggregate:
       - get: ci
+      - get: meta
```

Comment:
Several other things in this aggregate block are only used with certain tasks also. Also since this all run in parallel, adding things here doesn't slow anything down.